variables.tf

```diff
@@ -64,42 +64,42 @@ variable "cbr_rules" {
   default     = []
 }
 
 variable "skip_iam_authorization_policy" {
   type        = bool
-  description = "Set to true to skip the creation of an IAM authorization policy that permits all Event Notification instances in the resource group to read the encryption key from the KMS instance. No policy is created if var.kms_encryption_enabled is set to false."
+  description = "Set this variable value to `true` to skip the creation of an IAM authorization policy that permits all Event Notifications instances in the resource group to read the encryption key from the key management service (KMS) instance. No policy is created if `var.kms_encryption_enabled` variable is set to `false`."
   default     = false
 }
 
 variable "kms_encryption_enabled" {
   type        = bool
-  description = "Set this to true to control the encryption keys used to encrypt the data that you store in Event Notification. If set to false, the data is encrypted by using randomly generated keys. For more info on Managing Encryption, see https://cloud.ibm.com/docs/event-notifications?topic=event-notifications-en-managing-encryption"
+  description = "Set this variable value to `true` to control the encryption keys that are used to encrypt the data that you store in the Event Notifications instance. If this value is set to `false`, the data is encrypted by using randomly generated keys. For more information, see Managing encryption https://cloud.ibm.com/docs/event-notifications?topic=event-notifications-en-managing-encryption."
   default     = false
 }
 
 variable "existing_kms_instance_crn" {
-  description = "The CRN of the Hyper Protect Crypto Services or Key Protect instance. Required only if var.kms_encryption_enabled is set to true"
+  description = "The Cloud Resource Name (CRN) of the Hyper Protect Crypto Services or Key Protect instance. This variable is required only if `var.kms_encryption_enabled` is set to `true`."
   type        = string
   default     = null
 }
 
 variable "root_key_id" {
   type        = string
-  description = "The Key ID of a root key, existing in the KMS instance passed in var.existing_kms_instance_crn, which will be used to encrypt the data encryption keys (DEKs) which are then used to encrypt the data. Required if var.kms_encryption_enabled is set to true."
+  description = "The key ID of a root key, existing in the key management service (KMS) instance passed in `var.existing_kms_instance_crn`, which will be used to encrypt the data encryption keys (DEKs) which are then used to encrypt the data. This variable is required if `var.kms_encryption_enabled` is set to `true`."
   default     = null
 }
 
 
 variable "service_credential_names" {
-  description = "Map of name, role for service credentials that you want to create for the event notification"
+  description = "The mapping of names and roles for service credentials that you want to create for the Event Notifications instance."
```

Comment:
Would an example be helpful here?

Reply:
examples are ugly right now in the UI. Catalog doesn't handle the syntax well. Conall opened an issue for it.

Reply:
Let's ask what the expected schema would be here and provide maybe one example instead of all the accepted values shown below on line 103